src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/request/StrikeOffObjectionsEntity.java

```diff
@@ -0,0 +1,95 @@
+package uk.gov.companieshouse.api.strikeoffobjections.model.request;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import java.time.LocalDateTime;
+
+@Document(collection = "strike_off_objections")
+public class StrikeOffObjectionsEntity {
+
+
+    public static class Builder {
+        private String reason;
+        private CreatedBy createdBy;
+        private LocalDateTime createdOn;
+        private RequestStatus status;
+
+        public Builder withReason(String requestInformation) {
+            this.reason = requestInformation;
+            return this;
+        }
+
+        public Builder withCreatedBy(CreatedBy createdBy) {
+            this.createdBy = createdBy;
+            return this;
+        }
+
+        public Builder withCreatedOn(LocalDateTime createdOn) {
+            this.createdOn = createdOn;
+            return this;
+        }
+
+        public Builder withStatus(RequestStatus status) {
+            this.status = status;
+            return this;
+        }
+
+        public StrikeOffObjectionsEntity build() {
+            StrikeOffObjectionsEntity strikeOffObjectionsEntity = new StrikeOffObjectionsEntity();
+            strikeOffObjectionsEntity.setReason(this.reason);
+            strikeOffObjectionsEntity.setCreatedBy(this.createdBy);
+            strikeOffObjectionsEntity.setCreatedOn(this.createdOn);
+            strikeOffObjectionsEntity.setStatus(this.status);
+            return strikeOffObjectionsEntity;
+        }
+    }
+
+    @Id
+    private String id;
+    @Field("reason")
+    private String reason;
+    @Field("created_by")
+    private CreatedBy createdBy;
+    @Field("created_on")
+    private LocalDateTime createdOn;
+    @Field("status")
+    private RequestStatus status;
```

Comment:
I think this entity is missing 'companyNumber'.

Reply:
company number added.